spec/features/scheduling/scheduled_publishing_failed_spec.rb

```diff
@@ -17,8 +17,8 @@
 
     when_i_visit_the_summary_page
     then_i_can_see_the_publishing_failed
-    and_there_is_a_history_entry
     and_i_can_edit_the_edition
+    and_there_is_a_history_entry
```

Comment:
Should we update this to timeline_entry which I think is the terminology used on the other tests.

Reply:
Yup good point